frontend/tests/components/ProcessNext.test.tsx

```diff
@@ -11,4 +13,11 @@ describe("Process Content", () => {
 
     expect(ProcessH1).toBeInTheDocument();
   });
+  it("github link", () => {
```

Comment:
can you make this test more descriptive? maybe `gitHubIssueLink renders a component with correct props`? This could also be rewritten to look at the href on the node, which would be slightly more helpful

Reply:
I updated it